docs/resources/advanced_cluster.md

```diff
@@ -620,7 +618,7 @@ lifecycle {
 
 ~> **IMPORTANT:** If `compute_enabled` is true, then Atlas will automatically scale up to the maximum provided and down to the minimum, if provided.
 This will cause the value of `instance_size` returned to potentially be different than what is specified in the Terraform config and if one then applies a plan, not noting this, Terraform will scale the cluster back to the original `instance_size` value.
```

Comment:
Same here: replace "if one then applies a plan, " with "if you then apply a plan"